packages/react/src/components/Tile/Tile.js

```diff
@@ -503,29 +504,29 @@ export class ExpandableTile extends Component {
     const {
       tabIndex,
       className,
+      expanded, // eslint-disable-line
       tileMaxHeight, // eslint-disable-line
       tilePadding, // eslint-disable-line
       handleClick, // eslint-disable-line
       tileCollapsedIconText, // eslint-disable-line
       tileExpandedIconText, // eslint-disable-line
+      onBeforeClick, // eslint-disable-line
       light,
       ...other
     } = this.props;
 
-    const { expanded } = this.state;
```

Comment:
IMHO this is confusing because there is a prop called `expanded` already 😅 
So I suggest removing it and just referencing the state with `this.state.expanded` instead.

Reply:
Or you can do: `const { expanded: expandedState } = this.state`?

Reply:
Thanks @asudoh! I went ahead and updated the code per your suggestion.